src/Data/NonEmpty.purs

```diff
@@ -54,7 +60,7 @@ oneOf :: forall f a. (Alternative f) => NonEmpty f a -> f a
 oneOf (NonEmpty a fa) = pure a <|> fa
 
 instance showNonEmpty :: (Show a, Show (f a)) => Show (NonEmpty f a) where
-  show (NonEmpty a fa) = "(NonEmpty " ++ show a ++ " " ++ show fa ++ ")"
+  show (NonEmpty a fa) = "NonEmpty (" ++ show a ++ ") (" ++ show fa ++ ")"
```

Comment:
I think this is the way it should have been, isn't it? See `Either`, `Maybe`, etc.